lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
haha. i've added some commits + rebased. i've added the `time_without_date` to each locale file, which seems dumb, but adequate, so please let me know if you see room for improvement 👍

fyi i'm getting a few failed tests on master, and they remain failing on this branch:

```
rspec ./spec/features/sidebar_spec.rb:12 # sidebar displays translated name of model
rspec ./spec/features/show_page_spec.rb:105 # customer show page displays translated labels
rspec ./spec/i18n_spec.rb:9 # I18n does not have missing keys
rspec ./spec/features/index_page_spec.rb:50 # customer index page displays translated labels
rspec ./spec/features/form_spec.rb:12 # edit form displays translated labels
```